spec/sidekiq/extensions/manager_spec.rb

```diff
@@ -0,0 +1,17 @@
+RSpec.describe Sidekiq::Manager do
```

Comment:
I added this test to have a better coverage (because this would have failed when upgrading to Sidekiq v7)